src/bin/main.rs

```diff
@@ -100,12 +100,24 @@ fn make_opts() -> Options {
          with 1 and prints a diff if formatting is required.",
     );
     let is_nightly = is_nightly();
-    let emit_opts = if is_nightly {
-        "[files|stdout|coverage|checkstyle]"
+    let (emit_opts, emit_desc) = if is_nightly {
+        (
+            "[files|stdout|coverage|checkstyle]",
+            "Files writes output to files, Stdout writes output to stdout, Coverage displays \
+             how much of the input file was processed, Checkstyle shows unfancy stdout output.",
+        )
     } else {
-        "[files|stdout]"
+        (
+            "[files|stdout]",
+            "Files: writes output to files, Stdout: writes output to stdout",
+        )
     };
-    opts.optopt("", "emit", "What data to emit and how", emit_opts);
+    opts.optopt(
+        "",
+        "emit",
+        &format!("What data to emit and how. {}", emit_desc),
+        emit_opts,
+    );
```

Comment:
Am I missing something? If I ctrl-f the README for "emit", there are no matches. I think this is helpful new information.

In general I agree with not duplicating information (hard to keep copies in sync reliably), but I would advocate for shipping the information with the tool rather than expecting people using the tool to visit a website / clone the source repository to figure out how it works.

Reply:
I'm perhaps a bit sensitive to 'you can look it up online' for help -- I've been spending a lot of time lately in areas with very poor to no Internet availability and have come to greatly value "batteries included" utilities. I'm not adverse to having information in the README as well, but I think it's important for accessibility of tools for persons in less developed areas to ensure that if the tool doesn't require the Internet to run, it shouldn't require the Internet to use.